website_baidu_map/README.rst

```diff
@@ -0,0 +1,38 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=================
+website_baidu_map
+=================
+
+This module extends the functionality of website to support baidu map in the contact us page.
+
+Installation
+============
+
+To install this module, you need to:
+
+ * have basic modules installed (website, base)
+
+
```

Comment:
Known issues / Roadmap
====================
This snippet relies on the old Baidu API, to be upgraded.
Currently only works in http and generate a mixed content in https website.